src/main/java/bio/terra/cloudfunctions/common/MediaTypeWrapper.java

```diff
@@ -0,0 +1,50 @@
+package bio.terra.cloudfunctions.common;
```

Comment:
Can we use com.google.common.net.MediaType?

Reply:
There are multiple java implementations of Content-type enum, I found that `com.google.common.net.MediaType` is the most comprehensive and extensible. It does not currently support `gzip` or `json` content types so I extended them.

 ` // Workaround for Mime types not found in Guava MediaType
  public static final MediaType APPLICATION_GZIP = MediaType.create(MediaType.GZIP.type(), "gzip");
  public static final MediaType APPLICATION_JSON =
      MediaType.create(MediaType.JSON_UTF_8.type(), "json");`

Reply:
I see `public static final MediaType GZIP = createConstant(APPLICATION_TYPE, "x-gzip");` in Google class and something similar for JSON.  It might now match the content type string in the message, but I'm not sure if that warrants creating a whole new class like this.

Reply:
The content type string we received from the GCS object finalize messages are either `gzip` or `json`, haven't found a better alternative to extrapolate non-matching content type especially `MediaType` is a final class.